mailpoet/lib/API/MP/v1/API.php

```diff
@@ -111,9 +119,21 @@ public function unsubscribeFromLists($subscriberId, array $listIds) {
     return $subscriber->withCustomFields()->withSubscriptions()->asArray();
   }
 
-  public function getLists() {
-    return Segment::where('type', Segment::TYPE_DEFAULT)
-      ->findArray();
+  public function getLists(): array {
+    $segments = $this->segmentsRepository->findBy(['type' => SegmentEntity::TYPE_DEFAULT]);
```

Comment:
I guess this wasn't there previously either but should we add a default order here?